user/homework.go

```diff
@@ -57,6 +59,31 @@ func (hw *Homework) Save() error {
 	return nil
 }
 
+// SaveAfterFetch filters out problems for tracks the user has yet to start
+// before saving them all to disk.
+func (hw *Homework) SaveAfterFetch(numArgs int) error {
+	if numArgs == 0 {
+		var existingHW []*Item
+		for _, item := range hw.Items {
+			path := filepath.Join(item.dir, item.TrackID)
+			_, err := os.Stat(path)
+			if err == nil {
+				existingHW = append(existingHW, item)
+			}
+		}
+		hw.Items = existingHW
+		if len(hw.Items) == 0 {
+			fmt.Println(`
+You have yet to start a language track!
+View all available language tracks with "exercism tracks"
+Fetch exercises for your first track with "exercism fetch LANGUAGE"
+`)
+			os.Exit(0)
```

Comment:
A couple things (philosophically):

1. I think we should only use `os.Exit` for a value other than 0 (in the case of an error), and
2. I don't think we should ever have `os.Exit` in packages other than `cmd`.